src/common/reactNativeProjectHelper.ts

```diff
@@ -49,6 +51,18 @@ export class ReactNativeProjectHelper {
             });
     }
 
+    public static verifyVersion(version: string): string {
```

Comment:
I suppose that the range of use of this function may be wider than only telemetry. It can be necessary to verify a package version in some other places of the extension code in future. So I think that ReactNativeProjectHelper class could be more appropriate place for this function. Could you please explain why this funcion should be moved to TelemetryHelper?

Reply:
I am not sure that we need to use that function somewhere else because we don't need special handlers to transform incorrect version to a special string literals.

Reply:
Ok, I'll move this function to telemetry helper.